python/tank/log.py

```diff
@@ -557,9 +557,34 @@ def initialize_base_file_handler(self, log_name):
         :returns: The name of the previous log_name that is being switched away from,
                   None if no base logger was previously active.
         """
+        # avoid cyclic references
+        from .util import filesystem
+
+        return self.initialize_base_file_handler_from_path(
+            os.path.join(
+                self.log_folder,
+                "%s.log" % filesystem.create_valid_filename(log_name)
+            )
+        )
+
+    def initialize_base_file_handler_from_path(self, log_file):
+        """
+        Create a file handler and attach it to the sgtk base logger.
+
+        This method is there for legacy Toolkit applications and shouldn't be used. Use
+        ``initialize_base_file_handler`` instead.
+
+        :param log_file: Path of the file to write the logs to.
+
+        :returns: The name of the previous log_name that is being switched away from,
```

Comment:
Would it make sense that this method returned the full path, but the `initialize_base_file_handler` method would (as it's doing today) would return the log name? That feels more symmetrical to me.